src/xokdinst.rs

```diff
@@ -246,7 +257,37 @@ enum Opt {
     },
 }
 
-fn cmd_installer(version: Option<&str>) -> std::process::Command {
+/// Compute master memory: host memory - 6GB (4 for worker + 2 for bootstrap)
```

Comment:
This seems like it will fall over unless `--size single`.  IOW we need to account for the number of control plane nodes right?

Reply:
That is correct, new head should compute proper resource per clustersize.